app/javascript/dashboard/routes/dashboard/settings/agents/Index.vue

```diff
@@ -246,3 +246,8 @@ export default {
   },
 };
 </script>
+<style lang="scss" scoped>
+.content-spacing {
```

Comment:
@iamsivin This style is repeated in every component which is not required. Please move this to a common stylesheet.